utils/docker/Dockerfile.centos.7

```diff
@@ -9,3 +9,5 @@ MAINTAINER Brian J. Murrell <brian.murrell@intel.com>
 
 RUN yum install -y epel-release
 RUN yum install -y python-requests python-github perl ShellCheck git
+RUN yum install -y python-pip
+RUN pip install codespell
```

Comment:
Another package that needs a ticket for making an RPM of.  Can you open that please?  Did one get opened for the package you were installing using `pip` in a previous review I did?

Reply:
Same package, although this is codespell over python2, previously it was codespell over python3.  Once we've decided on the best approach then we can file tickets to support it.